text/0000-manifest-list-buildpacks.md

```diff
@@ -0,0 +1,159 @@
+# Publish multi-arch buildpacks
+
+## Summary
+
+In order to enable end users to create arm64 images with paketo buildpacks I am proposing that paketo start publishing multi-arch buildpack images for all buildpacks. @dmikusa put together [this](https://github.com/dmikusa/paketo-arm64) excellent guide for building arm64 containers with paketo buildpacks, but it puts a heavy burden on end users to create and maintain copies of paketo buildpacks. This proposal is to start publishing manifest list images that support x86_64 and arm64 architectures. This would be accomplished by using `docker buildx imagetools create` or similar.
+
+For simplicity I will refer to x86_64 as amd64 throughout this document.
+
+## Motivation
+
+The number one question I get about [paketo] buildpacks is whether they support arm. It is safe to say that workloads are generally moving to arm processors and end users are already evaluating how to build containers on arm. It is difficult for me as an end user to seriously consider building arm images with paketo buildpacks without putting in some engineering effort into building arm versions of the paketo buildpacks I may need. While the above example is definitely helpful, a lot of the steps can be built into the github actions workflows that create and release buildpacks for general consumption. I believe arm support was the number one requested roadmap item for 2023, and this proposal is an attempt to jump start that process.
+
+## Detailed Explanation
+
+Right now binaries are created on/for amd64 and then packaged up into buildpacks using pack. It should be easy to create binaries for arm64 which can then be packaged up into buildpacks using pack. These architecture-specific buildpacks can be pushed to a registry with a platform specific tag as shown in the examples below. 
+
+* `paketobuildpacks/ca-certificates:3.6-amd64`
+* `paketobuildpacks/ca-certificates:3.6-arm64`
```

Comment:
As I'm reading about `docker buildx`, a lot of the examples seem to push a single image with multiple tags from a single `Dockerfile`. It looks like `docker buildx imagetools create` does the same thing but with multiple source images that already exist on a registry.

Is it possible to avoid the intermediate step of publishing buildpacks to the `:<version>-<arch>` tag? It seems unnecessary once we've created a single manifest/image for all platforms. Could we create the images on a local registry and just push the single multi-arch image?

Reply:
Circling back on this, what's the outcome? Can we go for publishing the single multi-arch image?